lisa/tools/ntttcp.py

```diff
@@ -2,16 +2,76 @@
 # Licensed under the MIT license.
 
 import re
+from decimal import Decimal
 from typing import List, Type
 
 from lisa.executable import Tool
+from lisa.messages import (
+    NetworkNtttcpTCPPerformanceMessage,
+    NetworkNtttcpUDPPerformanceMessage,
+)
 from lisa.tools import Gcc, Git, Make
-from lisa.util.process import ExecutableResult
+from lisa.util.process import ExecutableResult, Process
+
+NTTTCP_TCP_CONCURRENCY = [
```

Comment:
Can they be merged with iperf constants? It doesn't need to take different test parameters for different tools. They can be put into a shared file for perf tools.

Reply:
they are different

Reply:
I know the tcp concurrency is not used in iperf, and I don't know why. But the udp concurrency is used in both. We may add other network perf tools in future. So it needs to manage the built-in parameters together.